Resources/Prototypes/_Goobstation/Changeling/Entities/Objects/Shields/shields.yml

```
@@ -0,0 +1,31 @@
+      state: ling-icon
+    - type: Item
+      sprite: _Goobstation/Changeling/shields.rsi
+      heldPrefix: ling
+    - type: Destructible
+      thresholds:
+        - trigger:
+            !type:DamageTrigger
+            damage: 60
+          behaviors:
+            - !type:DoActsBehavior
+              acts: [ "Destruction" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)